recipes/cgns/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(test_package test_package.cpp)
+target_link_libraries(test_package ${CONAN_LIBS})
```

https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md#verifying-components

```suggestion
find_package(CGNS CONFIG REQUIRED)

add_executable(test_package test_package.cpp)
if(SHARED)
    target_link_libraries(test_package CGNS::cgns_shared)
else()
    target_link_libraries(test_package CGNS::cgns_static)
endif()
```